### PR TITLE
Use helpers instead of URLs in switch checkboxes

### PR DESCRIPTION
**Pull request by differentreality**
**Checklist**

- [ ] I have read the [Contribution & Best practices Guide](https://github.com/openSUSE/osem/blob/master/CONTRIBUTING.md).
- [ ] My branch is up-to-date with the upstream `master` branch.
- [ ] The tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works(if appropriate).
- [ ] I have added necessary documentation (if appropriate).

**Short description of what this resolves/which [issues](https://github.com/openSUSE/osem/issues) does this fix?:**

- https://github.com/openSUSE/osem/issues/2472

**Changes proposed in this pull request:**

**Pending** 
- https://github.com/openSUSE/osem/blob/master/app/views/admin/users/index.html.haml#L42 
- registrations toggle attendance (which seems to be removed, but shouldn't?)

### CODE REVIEW

File: app/views/admin/conferences/_recent_users.html.haml [hunk 1]
```diff
@@ -15,7 +15,7 @@
             %td= user.created_at.strftime('%m/%d/%Y')
             %td
               = check_box_tag user.id, user.id, user.confirmed?,
-                url: "/admin/users/#{user.id}/toggle_confirmation?user[to_confirm]=",
+                url: "#{toggle_confirmation_admin_user_path(user.id)}?user[to_confirm]=",
```
Comment (Ana06) [inline app/views/admin/conferences/_recent_users.html.haml]: you can also add the parameters to the route: 
```ruby
toggle_confirmation_admin_user_path(3, user: { to_confirm: 'true' })
```

**Comment by differentreality** [comment]
@Ana06 what do you think?

File: app/views/admin/events/registrations.html.haml [hunk 2]
```diff
@@ -32,7 +32,7 @@
               %td= event_registration.email
               %td= event_registration.created_at
               %td
-                = check_box_tag @conference.short_title, @event.id, event_registration.attended, class: 'switch-checkbox', url: "/admin/conferences/#{@conference.short_title}/program/events/#{@event.id}/toggle_attendance?events_registration_id=#{event_registration.id}&&event_registration[attended]="
+                = check_box_tag @conference.short_title, @event.id, event_registration.attended, class: 'switch-checkbox', url: "#{toggle_attendance_admin_conference_program_event_path(@event.id)}?events_registration_id=#{event_registration.id}&&event_registration[attended]="
```
Comment (Ana06) [inline app/views/admin/events/registrations.html.haml]: this route is wrong, conference is missing!

**Comment by Ana06** [comment]
@differentreality I have made the requested changes and remove one of the checkboxes:

### Before

image

### After

image

I hope it is fine. :wink:

**Pull request merged**